versions/ic.ver

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Initial condition subfolder versions
+
+#shellcheck disable=SC2034
+declare -A ic_versions
+
+ic_versions['C48']=20241120
+ic_versions['C48mx500']=20241120
```

Comment:
Oh, the `icsdir` paths in the CI test yamls ([example](https://github.com/NOAA-EMC/global-workflow/blob/develop/ci/cases/pr/C48mx500_3DVarAOWCDA.yaml#L12)) will also need to be updated for the resolutions that have a new timestamp.

Reply:
We might want to update the title of this PR to reflect the expanded scope of the IC dirs that this touches to get more eyes on things that are changing. This PR is going beyond just wave ICs at this point - although the main reason for the PR, and the fix is greatly appreciated.

Reply:
Paths updated, though we probably don't need to specify the icsdir at all in these.